source/common/stats/stats_impl.cc

```diff
@@ -152,38 +166,72 @@ TagProducerImpl::TagProducerImpl(const envoy::config::metrics::v2::StatsConfig&
   }
 }
 
-std::string TagProducerImpl::produceTags(const std::string& stat_name,
-                                         std::vector<Tag>& tags) const {
-  tags.insert(tags.end(), default_tags_.begin(), default_tags_.end());
+void TagProducerImpl::addExtractorsMatching(absl::string_view name) {
+  int num_found = 0;
```

Comment:
Nope, you're probably right.  I just think it's preferable to use the explicitly sized integers everywhere  (especially in non-test code) rather than making a somewhat subjective exception for small values.  Not a big deal, and if you want to leave as is, that's fine.

Reply:
I don't feel strongly about this at all, but a recursive-grep through envoy source shows lots of existing 'int' usage.  Some of them are related to system-calls for that type, but there's plenty of others that look like they are counting small things.  So I'm inclined to keep this as 'int'.

Reply:
SGTM